npm install fails

It looks like it's because gulp doesn't have an install task, and the install script in package.json specifies gulp build && gulp install.

```
> catserver@1.0.0 install /Users/lewis/dev/catserver
> gulp build && gulp install

[13:23:32] Using gulpfile ~/dev/catserver/gulpfile.js
[13:23:32] Starting 'build-manifest'...
[13:23:32] Starting 'build-js'...
[13:23:32] Starting 'build-html'...
[13:23:32] Starting 'build-css'...
[13:23:32] Starting 'build-img'...
[13:23:32] Starting 'build-www'...
[13:23:32] Finished 'build-manifest' after 91 ms
[13:23:32] Finished 'build-html' after 78 ms
[13:23:32] Finished 'build-css' after 77 ms
[13:23:32] Finished 'build-www' after 119 ms
[13:23:32] Finished 'build-js' after 235 ms
[13:23:32] Finished 'build-img' after 227 ms
[13:23:32] Starting 'build'...
[13:23:32] Finished 'build' after 23 μs
[13:23:33] Using gulpfile ~/dev/catserver/gulpfile.js
[13:23:33] Task 'install' is not in your gulpfile
[13:23:33] Please check the documentation for proper gulpfile formatting

npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.5
npm ERR! code ELIFECYCLE
npm ERR! catserver@1.0.0 install: `gulp build && gulp install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the catserver@1.0.0 install script 'gulp build && gulp install'.
npm ERR! This is most likely a problem with the catserver package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp build && gulp install
npm ERR! You can get their info via:
npm ERR!     npm owner ls catserver
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lewis/dev/catserver/npm-debug.log
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -8,6 +8,7 @@ var replace = require('gulp-replace');
 var yargs = require('yargs');
 var path = require('path');
 var fs = require('fs');
+var shell = require('gulp-shell');
 
 var appPath = path.join(__dirname, 'build');
 
@@ -52,4 +53,8 @@ gulp.task('build-www', function() {
   return gulp.src('www/**/*')
     .pipe(gulp.dest('./build/www'));
 });
+
+gulp.task('install', shell.task([
+  'install-to-adb build/ --launch'
+]));
 

--- a/package.json
+++ b/package.json
@@ -32,6 +32,8 @@
     "gulp-jshint": "^1.10.0",
     "gulp-rename": "^1.2.2",
     "gulp-replace": "^0.5.3",
+    "gulp-shell": "^0.4.1",
+    "install-to-adb": "^1.1.0",
     "yargs": "^3.7.0"
   }
 }

